### PR TITLE
Adding definitons for Interconnect LTS 1.10.1

### PR DESCRIPTION
**Pull request by granzoto**
@fgiorgetti could you please approve and merge ? 
Signed-off-by: Renato Granzoto <rgranzot@redhat.com>

### CODE REVIEW

File: 1.10.1/1-subtask_common_tasks.csv [hunk 1]
```diff
@@ -0,0 +1,5 @@
+PARENT_TASK,SUMMARY,ESTIMATE,DESCRIPTION
+JIRAID,Create mrgrepo entry for 1.10.1,2h,Run mrgjenkins job to create the mrgrepo entry.\nSee the document below for further reference: https://docs.google.com/document/d/1dIjdSdU0sIZ7QVQLcgysiIwm-BJJZ83mePyykbm11Dw/edit
+JIRAID,Adjust job dsl on amq-ci to consume latest image tag 1.10,4h,Adjust job dsl on amq-ci to consume latest image tag 1.10
+JIRAID,Clean up old and broken VMs in PSI,4h,Go through VMs and remove those that were left behind and those in error state
+JIRAID,Remove old OCP cluster resources from PSI,1d,Remove resources for old OCP clusters that are broken or were left behind
```
Comment (fgiorgetti) [inline 1.10.1/1-subtask_common_tasks.csv:5]: I believe this is not needed now.

File: 1.10.1/4-subtask_test_devel.csv [hunk 2]
```diff
@@ -0,0 +1,2 @@
+PARENT_TASK,SUMMARY,ESTIMATE,DESCRIPTION
+JIRAID,Run qdr-image test suite against OpenShift,2d,Run qdr-image test suite against OpenShift (current and current-1)
```
Comment (fgiorgetti) [inline 1.10.1/4-subtask_test_devel.csv:2]: You can remove this file `1.10.1/4-subtask_test_devel.csv`. It does not contain any test devel activity.

File: 1.10.1/tasks.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+       "customfield_12311240": {
+         "name": "1.10.1.GA"
+       },
+       "assignee": {
+          "name": ""
+       },
+       "issuetype": {
+          "name": "Task"
+       },
+       "labels": [
```
Comment (fgiorgetti) [inline 1.10.1/tasks.json:18]: Please add this field into the tasks.json:
```
       "security": {
          "id": "10291"
       }
```

**Review by fgiorgetti — CHANGES_REQUESTED** [review]

File: 1.10.1/tasks.json (hunk 3 above)
Comment (granzoto) [reply 1.10.1/tasks.json:21]: added

**Review by fgiorgetti — APPROVED** [review]

**Pull request merged**